int64 xmin
int64 xmax
int64 ymin 
int64 ymax
string clase
